ChatRPG/ChatRPG/API/OpenAiLlmClient.cs

```diff
@@ -0,0 +1,101 @@
+using Microsoft.IdentityModel.Tokens;
+using RestSharp;
+using RestSharp.Authenticators;
+
+namespace ChatRPG.API;
+
+public class OpenAiLlmClient : IOpenAiLlmClient, IDisposable
+{
+    private const string OpenAiBaseUrl = "https://api.openai.com/v1/";
+    private const string Model = "gpt-3.5-turbo";
+    private const double Temperature = 0.7;
```

Comment:
Hvad er Temperature her?

Reply:
- Temperature is a parameter that controls the “creativity” or randomness of the text generated by GPT-3. A higher temperature (e.g., 0.7) results in more diverse and creative output, while a lower temperature (e.g., 0.2) makes the output more deterministic and focused.

- In practice, temperature affects the probability distribution over the possible tokens at each step of the generation process. A temperature of 0 would make the model completely deterministic, always choosing the most likely token.

Source: [OpenAI](https://community.openai.com/t/cheat-sheet-mastering-temperature-and-top-p-in-chatgpt-api-a-few-tips-and-tricks-on-controlling-the-creativity-deterministic-output-of-prompt-responses/172683)